idl/cherami/cherami.thrift

```diff
@@ -550,7 +550,8 @@ struct ReconfigureInfo {
 
 enum InputHostCommandType {
   ACK,
-  RECONFIGURE
+  RECONFIGURE,
+  DRAIN // to trigger draining of a all connections to this inputhost
```

Comment:
yeah.. This is for *this* connection stream and from input we send the reconfigure on all connection streams.. client upon receiving this will drain the connections to this inputhost on *this* publisher object..